packages/react-components/react-progress/MIGRATION.md

```diff
@@ -0,0 +1,26 @@
+# Progress Migration
```

Comment:
IMO a better place for this documentation is in the [v9 doc site](https://github.com/microsoft/fluentui/tree/master/apps/public-docsite-v9/src/Concepts/Migration/FromV8/Components) but this is not a blocker for me.

To me, we should minimize the number of places documentation lives and the doc site is much more visible to our users that a Markdown file.